commands/uploader.go

```diff
@@ -332,13 +332,16 @@ func (c *uploadContext) Await() {
 }
 
 var (
+	githubHttps, _ = url.Parse("https://github.com")
+	githubSsh, _   = url.Parse("ssh://github.com")
```

Comment:
> Not totally sure on this one, since I think the :123 port number would be a part of the u.Host and u.Port() member/function, but wouldn't appear in the u.Path.

Correct you are! Ignoooore meeeee.